code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm

```diff
@@ -50,7 +53,8 @@
 		//It's also extremely buggy visually, so it's balance+bugfix
 		return
 
-	else //Maybe uses plasma in the future, although that wouldn't make any sense...
+	else // Added plasma cost for balance
+		plasma_cost = 15

```

Comment:
Yea thats returning an error but I dont understand why. Drone/Praetorian/Larva files all reference plasma_cost without defining it. What's the issue here?